src/frontend/src/CustomNodes/hooks/use-check-code-validity.tsx

```diff
@@ -14,7 +14,7 @@ const useCheckCodeValidity = (
     // first check if data.type in NATIVE_CATEGORIES
     // if not return
     if (
-      !NATIVE_CATEGORIES.includes(types[data.type]) ||
+      !Object.keys(nodeNames).includes(types[data.type]) ||
```

Comment:
@lucaseduoli is this change backwards compatible with existing flows? 
For example, if I'm using a compoenent without the "name" property, would the refresh button appear ?

Reply:
Yes, it is compatible. That line specifically just takes the types of the categories that we display at the sidebar instead of a types constant.